internal/pkg/cli/flag.go

```diff
@@ -95,7 +96,7 @@ const (
 	nameFlagShort = "n"
 	appFlagShort  = "a"
 	envFlagShort  = "e"
-	svcFlagShort  = "s"
+	wkldFlagShort = "w"
```

Comment:
Hmm instead of adding a `-w` and `--wkld` flag, what do you think of adding a `--for` flag instead? So that it reads:
```
copilot storage init --name my-bucket --type S3 --for frontend
```
My slight concern is that `wkld` not as easy to read or type, instead I'd prefer `--workload`  for the long name. The slight advantage of `--for` is that it doesn't introduce a new term in the help menu. So let me know your thoughts :)!

Reply:
this is also a breaking change for people relying on the `-s` short flag, can we add a "!" in the title of the PR to denote the change.

Reply:
I really like the `--for` syntax here. It's much more natural than even using `--svc` or `--job`.